lib/tasks/migrate_role_content.rake

```diff
@@ -0,0 +1,16 @@
+namespace :migrate do
```

Comment:
I think the convention is that usually the name of the namespace matches the name of the file. I also wonder if this could fit in [the existing `data_hygiene` namespace](https://github.com/alphagov/whitehall/blob/master/lib/tasks/data_hygiene.rake)?